tests/applications.spec.ts

```diff
@@ -67,6 +67,8 @@ describe("Applications", () => {
                     "countryCode": "1",
                     "number": "1555555578"
                 },
+                "nationality": "US",
```

Comment:
Can you remove the update to this test? If there are compilation issues resolve them using type assertions. Reason is that this is a test that runs on Unit's CI.

Reply:
Gotcha!